src/components/hwWalletLogin/index.js

```diff
@@ -6,6 +6,7 @@ import HardwareWalletLogin from './hwWalletLogin';
 const mapStateToProps = state => ({
   // TODO update this when isHarwareWalletConnected is refactored and we have devices in store
   devices: state.settings.isHarwareWalletConnected ? [{}] : [],
+  isAppOpen: state.settings.isHarwareWalletConnected && state.settings.isAppOpen,
```

Comment:
I would put `isAppOpen` already into the `devices` list, as it will be less changes when we will put devices in store:
```suggestion
  devices: state.settings.isHarwareWalletConnected ? [{ isAppOpen: state.settings.isAppOpen }] : [],
```